app/main.py

```diff
@@ -2,24 +2,76 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: str, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: str, owner: str) -> int:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: str, value: str) -> None:
+        if not isinstance(value, int):
+            raise TypeError("Must be an integer")
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError(
+                f"Value should be between"
+                f" {self.min_amount} and {self.max_amount}."
+            )
+
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+   def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+       self.name = name
+       self.age = age
+       self.weight = weight
+       self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+
+    age = IntegerRange(0, 0)
```

Comment:
The `IntegerRange` instances in `SlideLimitationValidator` should have non-zero ranges to be meaningful. Currently, they are initialized with (0, 0), which will not allow any values.